### ISSUE TITLE
showing unexpected error

### ISSUE THREAD

**Issue by Subhrajyoti987** [microsoft/vscode#185304]
Type: <b>Bug</b>

whenever I am using cin and cout in c++ it is showing error. I have to close vscode and open it again in order to remove this unexpected error.

VS Code version: Code 1.79.2 (695af097c7bd098fbf017ce3ac85e09bbc5dda06, 2023-06-14T08:57:04.379Z)
OS version: Windows_NT x64 10.0.22621
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-10300H CPU @ 2.50GHz (8 x 2496)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|7.91GB (1.44GB free)|
|Process Argv|--crash-reporter-id 13194aad-f6bb-4811-9e26-3508475dc861|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (33)</summary>

Extension|Author (truncated)|Version
---|---|---
better-comments|aar|3.0.2
arepl|alm|2.0.5
code-gnu-global|aus|0.2.2
doxdocgen|csc|1.4.0
competitive-programming-helper|Div|5.11.0
python-environment-manager|don|1.0.4
python-extension-pack|don|1.7.0
vscode-toggle-column-selection|eri|1.0.6
prettier-vscode|esb|9.13.0
file-icons|fil|1.0.29
code-runner|for|0.12.0
better-cpp-syntax|jef|1.17.2
cmake-language-support-vscode|jos|0.0.9
vsc-python-indent|Kev|1.18.0
lightrunplugin-saas|lig|1.12.0
vscode-dotnet-runtime|ms-|1.6.0
python|ms-|2023.10.1
vscode-pylance|ms-|2023.6.20
jupyter|ms-|2023.5.1001582324
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.0.15
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
cmake-tools|ms-|1.14.33
cpptools|ms-|1.16.0
cpptools-extension-pack|ms-|1.3.0
color-highlight|nau|2.5.0
autodocstring|njp|0.6.1
LiveServer|rit|5.7.9
cmake|twx|0.0.17
intellicode-api-usage-examples|Vis|0.2.7
vscodeintellicode|Vis|1.2.30
jinja|who|0.0.8

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30256860
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30410667
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
pynewext54:30695312
azure-dev_surveyone:30548225
3biah626:30602489
pyind779:30671433
89544117:30613380
pythonsymbol12:30671437
2i9eh265:30646982
showlangstatbar:30737416
vsctsb:30748421
pythonms35:30701012
03d35959:30757346
pythonfmttext:30731395
pythoncmv:30756943
fixhidewlkth:30730051
hideindicator:30766889
pythongtdpath:30769146
dh2dc718:30763024
pythonidxpt:30768921

```

</details>

**Comment by sean-mcmanus** [comment]
@codecreed20 This was from the cpptools extension -- it's fixed with the latest 1.16.3 release.

@chrmarti This was an important regression caused by the 1.16.0 (pre-release) of our cpptools extension. If any C++ issues from users with the cpptools extension installed could be moved to https://github.com/Microsoft/vscode-cpptools/issues that would be great.

**Comment by isidorn** [comment]
@sean-mcmanus sorry about that! Closing since this issues is for the C++ team. I am discussing with Bob how can we improve this process in the future.
